bin/docker-dev-shell

```diff
@@ -58,7 +58,7 @@ else
 fi
 
 set +e
-RUNNING=$(docker ps --format '{{.Names}}' | grep "$CONTAINER_NAME")
+RUNNING=$(docker ps --format '{{.Names}}' | grep "$CONTAINER_NAME$")
```

Comment:
Is the grep match intended to match the whole string, or are we intending to only match a line ending with the container name (which seems a bit odd)?

The `-w` option can be used to match exact strings.

Reply:
I should read the whole description. Already answered, sorry! 🙃

Reply:
Good point, I recall running into a weird outcome with `-w` but let me check.